src/constants.py

```diff
@@ -3,67 +3,11 @@
 """constants for slurmrestd-operator."""
 from pathlib import Path
 
-SLURMRESTD_GROUP_GID = 64031
-SLURMRESTD_USER_UID = 64031
-SLURMRESTD_USER_NAME = "slurmrestd"
-SLURMRESTD_GROUP_NAME = "slurmrestd"
+SNAP_COMMON = Path("/var/snap/slurm/common")
 
-MUNGE_KEY_PATH = Path("/etc/munge/munge.key")
+SLURMRESTD_GROUP_GID = 584788
+SLURMRESTD_USER_UID = 584788
+SLURMRESTD_USER_NAME = "snap_daemon"
+SLURMRESTD_GROUP_NAME = "snap_daemon"
```

Comment:
Are these constants used anywhere anymore? A quick search seems to show that all references were removed when the deprecated methods were culled on the legacy operations manager.